src/components/single-place/single-place.js

```diff
@@ -25,14 +25,17 @@ export default class SinglePlace extends Component {
 
   render() {
     const loading = (!this.state.place ? false : true);
+
+    const { match: { params } } = this.props;
```

Comment:
Please don't duplicate code. A few lines above we already have get id from url logic. You can get place id from "place" state in single place component.